video/out/hwdec/hwdec_cuda.c

```diff
@@ -75,23 +75,42 @@ static int cuda_init(struct ra_hwdec *hw)
     struct cuda_hw_priv *p = hw->priv;
     CudaFunctions *cu;
     int level = hw->probing ? MSGL_V : MSGL_ERR;
+    bool initialized = false;
+
+    inline int backend_init(void)
```

Comment:
Nested functions are a gcc extension that are not part of the language spec. This will fail to build with clang or on Windows.

https://gcc.gnu.org/onlinedocs/gcc/Nested-Functions.html

Reply:
Fixed with https://github.com/mpv-player/mpv/pull/14028/commits/693ec4af66a92c6934f9119161f405a84019f3fe

Forgot C didn't have nested functions...